core/dbt/main.py

```diff
@@ -639,7 +641,9 @@ def _build_list_subparser(subparsers, base_subparser):
         aliases=['ls'],
     )
     sub.set_defaults(cls=ListTask, which='list', rpc_method=None)
-    resource_values = list(ListTask.ALL_RESOURCE_VALUES) + ['default', 'all']
+    resource_values: List[str] = [
+        str(s) for s in ListTask.ALL_RESOURCE_VALUES
+    ] + ['default', 'all']
```

Comment:
mypy doesn't know that StrEnums are also strs. Which is pretty reasonable, I guess.